File: app/src/main/java/com/vmb/financialcalculator/DepositResultActivity.java
package com.vmb.financialcalculator;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class DepositResultActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_deposit_result);

        Bundle arguments = getIntent().getExtras();
        Deposit deposit = (Deposit) arguments.getSerializable(Deposit.class.getSimpleName());

        TextView total = findViewById(R.id.textTotal);
        total.setText(String.format("%,d", (long)deposit.getTotal()));

        TextView profit = findViewById(R.id.textProfit);
        profit.setText(String.format("%,d", (long)deposit.getProfit()));

        TextView additional_sum_total = findViewById(R.id.textAdditionalSumTotal);
        additional_sum_total.setText(String.format("%,d", (long)deposit.getAdditional_sum_total()));
    }
}

File: app/src/main/java/com/vmb/financialcalculator/DepositResultActivity.kt
package com.vmb.financialcalculator

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView

class DepositResultActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_deposit_result)
        val arguments = intent.extras
        val deposit = arguments!!.getSerializable(Deposit::class.java.simpleName) as Deposit?
        val total = findViewById<TextView>(R.id.textTotal)
        total.text = String.format("%,d", deposit!!.total.toLong())
        val profit = findViewById<TextView>(R.id.textProfit)
        profit.text = String.format("%,d", deposit.profit.toLong())
        val additional_sum_total = findViewById<TextView>(R.id.textAdditionalSumTotal)
        additional_sum_total.text = String.format("%,d", deposit.additional_sum_total.toLong())
    }
}